kairon/shared/data/data_objects.py

```
@@ -1129,4 +1129,13 @@ class POSIntegrations(Auditlog):
-    meta = {"indexes": [{"fields": ["bot", "provider"]}]}
\ No newline at end of file
+    meta = {"indexes": [{"fields": ["bot", "provider"]}]}
+
+
+class StorePageMetadata(Document):
+    bot = StringField(required=True)
+    user = StringField(required=True)
+    timestamp = DateTimeField(default=datetime.utcnow)
+    config = DictField()
+
+    meta = {"indexes": [{"fields": ["bot"]}]}
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)